src/web/mjs/connectors/azoraworld.mjs

```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class AzoraWorld extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'azoraworld';
+        super.label = 'AzoraWorld';
+        this.tags = [ 'webtoon', 'turkish', 'manga' ];
```

Comment:
Isn't it an `arabic` website?

Reply:
Aha yeah u're right, I messed up X)
I'm gonna change it.

Reply:
I changed it, that's good.